.github/workflows/docs.yaml

```diff
@@ -33,14 +37,15 @@ jobs:
         uses: actions/cache@v3
         with:
           path: |
+            ~/.npm
```

Comment:
This line shortens the installation time from 20s to 4s. It is not shown in the runtime chart because we currently always use the first run for each commit. This is fixed in https://github.com/stoat-dev/stoat/pull/92. After that PR is merged, the shortened runtime should show up in the chart.